packages/uma-auth-client/src/steps/ConnectUma.tsx

```diff
@@ -31,7 +33,7 @@ export const ConnectUma = () => {
       // const discoveryDocument = await fetch(`https://${umaDomain}/.well-known/uma-configuration`);
```

Comment:
Any reason not to just do this now? Is it not working? One cool note that just occurred to me is that the vasp should include it's supported commands in that config file too so if the client app needs some method that's not supported, we can fail before any other requests. https://j.pinkdrink.app/.well-known/uma-configuration

Reply:
Ah i was having trouble hitting the local pink drink discovery document so I was just doing this to quickly test.